cordapp-cpk2/src/main/kotlin/net/corda/plugins/cpk2/DependencyCalculator.kt

```diff
@@ -158,6 +158,10 @@ open class DependencyCalculator @Inject constructor(objects: ObjectFactory) : De
             disallowChanges()
         }
 
+        for (library in _libraries) {
+            logger.info("CorDapp library dependency: {}", library.name)
+        }
```

Comment:
I think we can chain these staements together:
```kotlin
_libraries.apply {
    setFrom(packageFiles - bundledFiles)
    disallowChanges()
}.forEach { library ->
    logger.info("CorDapp library dependency: {}", library.name)
}
```

Reply:
Done